traffic_portal/app/src/common/modules/table/servers/TableServersController.js

```diff
@@ -17,405 +17,633 @@
+	const columns = [
+		{
+			headerName: "Cache Group",
+			field: "cachegroup",
+			hide: false,
+			searchable: true
+		},
+		{
+			headerName: "CDN",
+			field: "cdnName",
+			hide: false,
+			searchable: true
+		},
+		{
+			headerName: "Domain",
+			field: "domainName",
+			hide: false,
+			searchable: true
+		},
+		{
+			headerName: "Host",
+			field: "hostName",
+			hide: false,
+			searchable: true
+		},
+		{
+			headerName: "HTTPS Port",
+			field: "httpsPort",
+			hide: true,
+			searchable: false,
+			filter: "agNumberColumnFilter"
+		},
+		{
+			headerName: "ID",
+			field: "id",
+			hide: true,
+			searchable: true,
+			filter: "agNumberColumnFilter"
+		},
+		{
+			headerName: "ILO IP Address",
+			field: "iloIpAddress",
+			hide: true,
+			searchable: true,
+			cellRenderer: "sshCellRenderer",
+			onCellClicked: null
+		},
+		{
+			headerName: "ILO IP Gateway",
+			field: "iloIpGateway",
+			hide: true,
+			searchable: false,
+			cellRenderer: "sshCellRenderer",
+			onCellClicked: null
+		},
+		{
+			headerName: "ILO IP Netmask",
+			field: "iloIpNetmask",
+			hide: true,
+			searchable: false
+		},
+		{
+			headerName: "ILO Username",
+			field: "iloUsername",
+			hide: true,
+			searchable: false
+		},
+		{
+			headerName: "Interface Name",
+			field: "interfaceName",
+			hide: true,
+			searchable: false
+		},
+		{
+			headerName: "IPv6 Address",
+			field: "ipv6Address",
+			hide: false,
+			searchable: true
+		},
+		{
+			headerName: "IPv6 Gateway",
+			field: "ipv6Gateway",
+			hide: true,
+			searchable: false
+		},
+		{
+			headerName: "Last Updated",
+			field: "lastUpdated",
+			hide: true,
+			searchable: false,
+			filter: "agDateColumnFilter",
+			valueFormatter: dateCellFormatter
+		},
+		{
+			headerName: "Mgmt IP Address",
+			field: "mgmtIpAddress",
+			hide: true,
+			searchable: false
+		},
+		{
+			headerName: "Mgmt IP Gateway",
+			field: "mgmtIpGateway",
+			hide: true,
+			searchable: false,
+			filter: true,
+			cellRenderer: "sshCellRenderer",
+			onCellClicked: null
+		},
+		{
+			headerName: "Mgmt IP Netmask",
+			field: "mgmtIpNetmask",
+			hide: true,
+			searchable: false,
+			filter: true,
+			cellRenderer: "sshCellRenderer",
+			onCellClicked: null
+		},
+		{
+			headerName: "Network Gateway",
+			field: "ipGateway",
+			hide: true,
+			searchable: true,
+			filter: true,
+			cellRenderer: "sshCellRenderer",
+			onCellClicked: null
+		},
+		{
+			headerName: "Network IP",
+			field: "ipAddress",
+			hide: false,
+			searchable: true,
+			filter: true,
+			cellRenderer: "sshCellRenderer",
+			onCellClicked: null
+		},
+		{
+			headerName: "Network MTU",
+			field: "interfaceMtu",
+			hide: true,
+			searchable: false,
+			filter: "agNumberColumnFilter"
+		},
+		{
+			headerName: "Network Subnet",
+			field: "ipNetmask",
+			hide: true,
+			searchable: false
+		},
+		{
+			headerName: "Offline Reason",
+			field: "offlineReason",
+			hide: true,
+			searchable: false
```

Comment:
why does it say some are not searchable? looks like they are are all searchable once they are visible.

Reply:
those don't belong there. I actually already got rid of those properties by the time you made this review :P